--- origsrc/jrtplib-3.7.1/configure.in	2007-05-14 07:46:56.000000000 -0500
+++ src/jrtplib-3.7.1/configure.in	2008-06-24 21:18:43.750000000 -0500
@@ -30,7 +30,8 @@
 dnl ---------------------------------------------------------------------------
 dnl Set language to C++ and do libtool stuff
 dnl ---------------------------------------------------------------------------
-
+AC_PROG_CC
+AC_PROG_CXX
 AC_LANG(C++)
 
 AM_PROG_LIBTOOL
--- origsrc/jrtplib-3.7.1/examples/Makefile.am	2006-06-07 04:10:04.000000000 -0500
+++ src/jrtplib-3.7.1/examples/Makefile.am	2008-06-24 21:17:34.593750000 -0500
@@ -6,6 +6,6 @@
 example4_SOURCES = example4.cpp
 example5_SOURCES = example5.cpp
 
-INCLUDES = @RTP_JTHREADINCLUDES@ -I ../src/
+INCLUDES = @RTP_JTHREADINCLUDES@ -I../src -I$(top_srcdir)/src
 LDADD = ../src/.libs/libjrtp.a @RTP_LINKLIBS@
 
--- origsrc/jrtplib-3.7.1/src/Makefile.am	2006-07-17 03:06:40.000000000 -0500
+++ src/jrtplib-3.7.1/src/Makefile.am	2008-06-24 21:17:34.609375000 -0500
@@ -15,7 +15,7 @@
 		     rtpcollisionlist.cpp rtpipv6address.cpp \
 		     rtpudpv6transmitter.cpp rtptimeutilities.cpp \
 		     extratransmitters/rtpfaketransmitter.cpp
-libjrtp_la_LDFLAGS = -release @VERSION@ @RTP_LINKLIBS@
+libjrtp_la_LDFLAGS = -release @VERSION@ -no-undefined @RTP_LINKLIBS@
 libjrtpinclude_HEADERS = rtcpapppacket.h rtcpbyepacket.h rtcpcompoundpacket.h \
 			 rtcpcompoundpacketbuilder.h rtcppacket.h rtcppacketbuilder.h \
 			 rtcprrpacket.h rtcpscheduler.h rtcpsdesinfo.h rtcpsdespacket.h \
